index.src.html

```diff
@@ -606,6 +608,50 @@ <h4 id="referrer-policy-delivery-implicit-workers">
 </section>
 
 <section>
+  <h2 id="element-interface-extensions">Element interface extensions</h2>
+
+  <h3 id="html-anchor-element">HTMLAnchorElement</h3>
+  <pre class="idl">
+    partial interface HTMLAnchorElement {
+      attribute DOMString referrerPolicy;
+    };
+  </pre>
+  <h4 id="html-anchor-element-attributes">Attributes</h4>
+  The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
+  <a>settings object</a>'s referrer policy, <a>limited to only known values</a>.
```

Comment:
This doesn't make sense. The IDL attribute must reflect the `referrerpolicy` content attribute.